lib/runtime-core/src/backing.rs

```diff
@@ -237,9 +237,9 @@ impl LocalBacking {
                     let table = &tables[local_table_index];
 
                     if (table.size() as usize) < init_base + init.elements.len() {
-                        let delta = (init_base + init.elements.len()) - table.size() as usize;
-                        // Grow the table if it's too small.
-                        table.grow(delta as u32).expect("couldn't grow table");
```

Comment:
Just double-checking. We don't want to grow the table in this case, right?

Reply:
Yes, not to grow appears to be the correct implementation. I checked with one other implementation, the minimum table size should be enough to fit the elems intializer.